components/radio/common/fn_getRadioModInUse.sqf

```diff
@@ -0,0 +1,17 @@
+params ["_side"];
+
+if (["acre_sys_radio"] call ace_common_fnc_isModLoaded) then 
+{
+	"ACRE"
+}
+else
+{
+	if (["task_force_radio"] call ace_common_fnc_isModLoaded) then 
```

Comment:
I don't really think we'll be switching to TFAR at this point but structuring the radio component as "mod agnostic" on the outside givs us leeway in case we do - the ACRE2 devs might fuck up or move on or something and then we can re-engineer the inside and leave outside references to `common` in place.

Reply:
Right, so more for abstraction - that's good then. 👍